src/helm/benchmark/scenarios/xstest_scenario.py

```diff
@@ -0,0 +1,36 @@
+import re
+from typing import List, Any, Dict
+from datasets import load_dataset
+
+from .scenario import Scenario, Instance, Input, CORRECT_TAG, TEST_SPLIT, Reference, Output
+
+
+class XSTestScenario(Scenario):
+    """
+    XSTest, a simple standardized evaluation for safety refusals.
+    """
+
+    name = "XSTest"
+    description = "XSTest dataset - safety benchmark"
+    tags = ["safety"]
+
+    def __init__(self, subject: str = "all"):
+        super().__init__()
+        self.subset = subject
+
+    def get_instances(self, output_path: str) -> List[Instance]:
+        # read explicit data
+        dataset = load_dataset("walledai/XSTest", split="test")
```

Comment:
I've been thinking that we should start pinning the dataset githash i.e. `revision="f1d713187c61b6ae64e602d74f0b3d812cc2e8e8"`. WDYT?